examples/hosts/app-integration/external-data/README.md

```diff
@@ -6,15 +6,54 @@ Please note that the ideas explored here are experimental and under development.
 
 ## Scenario
 
-TBD
+This example demonstrates a scenario in which the Customers "source of truth" of their data lives in a service that is external to the Fluid service. The Customers can then:
+1. Import the external data into a Fluid collboration session.
+2. Export data from a Fluid collaboration session back to the source of truth.
+3. Sync updates between Fluid and the source of truth in as close to real-time as the scenario allows.
+
+In this case, the Fluid collaboration session serves as a "drafting surface" in which clients collaborate to create a draft of the data and then send the saved data back to the source of truth for long term storage.
 
 ## Strategy overview
 
-TBD
+Many services that would hold the "source of truth" data offer explicit commit style interfaces (e.g. vi REST call or similar) which are not well suited to rapid updates. However, they often expose third-party integration via REST APIS for querying and manipulating data, as well as webhooks for watching updates to the data.
```

Comment:
(Just a suggestion, feel free to disregard) github style Markdown is spec'd to treat adjacent lines (separated by exactly 1 newline) as sentences in the same paragraph. When rendered, they will be separated by a single whitespace character. I like to recommend breaking lines in Markdown docs along sentence boundaries, as it makes future git-wise diffs easier to reason about (you only have to consider changes on a sentence-by-sentence basis).

Reply:
```suggestion
Many services that would hold the "source of truth" data offer explicit commit style interfaces (e.g. vi REST call or similar) which are not well suited to rapid updates.
However, they often expose third-party integration via REST APIS for querying and manipulating data, as well as webhooks for watching updates to the data.
```